### PR TITLE
Add user-facing documentation

### PR DESCRIPTION
**Pull request by remcopeereboom**
Added ABOUT.md and LEARNING.md.

### CODE REVIEW

**Comment by snahor** [comment]
Looks good! but, I feel kinda confused, sometimes you use ML when I think you mean SML.

File: docs/ABOUT.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+developed in the early 1970s at the University of Edinburgh. Despite it's age, ML feels very young in many ways.
+ML had features that mainstream languages would not pick up for decades and are still being experimented with today.
+Here are some of ML's "cutting-edge" features:
+
+- [**strong** static typing](https://en.wikipedia.org/wiki/Type_system#Static_typing)
+- [automatic type inference](https://en.wikipedia.org/wiki/Type_inference)
+- exception handling
+- [pattern matching](https://en.wikipedia.org/wiki/Pattern_matching)
+- parametric polymorphism
+- [irst class functions](https://en.wikipedia.org/wiki/First-class_function)
```
Comment (yurrriq) [inline docs/ABOUT.md]: "irst" should be "first"

**Comment by m-a-ge** [comment]
Hm, maybe it's possible to merge this one since adds real value?

**Comment by mcmillhj** [comment]
I believe that some changes were requested but the original author never followed up. I'll update it and try to get this in sometime today to help out with https://github.com/exercism/meta/issues/85

**Pull request merged**